sjb/hack/determine_install_upgrade_version.py

```diff
@@ -80,8 +90,9 @@ def determine_search_version(pkg_name, pkg_version):
 	yb.conf.disable_excludes = ["all"]
```

Comment:
Unrelated to this PR but we should check to see if we can remove this -- was this due to some excluder bug with them being on all the time or are we always running this script when they are on and we need to keep it?